example-deployment/README.md

```diff
@@ -0,0 +1,267 @@
+# About
+
+Unlike a traditional ROS package, a webapp is usually deployed along with many other components. Here is an example of how a web deployment might looks like
+![](https://www.fullstackpython.com/img/visuals/full-stack-python-map.png)
+Credits: https://www.fullstackpython.com
+
+`rmf-web` only provides the application, as you can see in the diagram, a full web deployment consists of many, many other components. There is no single way to "run" `rmf-web` as you would normally expect from a desktop application. Each deployment will be different, depending on the environment that you are running, the licenses you can use, the SaaS that you are using, the cloud infrastructure (or lack of) and many other factors, and the application itself is only one of the component among the sea of many software used in a modern webapp deployment.
+
+This repo serves as an example deployment of `rmf-web` that can work in an airgapped network. Meaning that we will not be using any SaaS and public cloud infrastructure.
+
+# Software Stack
+
+In this example, we will be using the following software
+
+rmf-web:
+* ros2-bridge
+* dashboard
+
+others:
+* kubernetes
+* minikube
```

Comment:
Just to check (maybe for the future), and maybe I'm wrong, but since we will also want to use Fluentd and use it in an air-gapped closed env,  isn't something like rancher a better fit for this?

Reply:
Awesome documentation btw. I'll try to test it tomorrow

Reply:
yea, rancher will better in general. I used minikube because it is the only "quick k8s" solution I have experience with.